test/mappers/http-error-mapper.test.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const { HttpError } = require('@uphold/http-errors');
+const HttpErrorMapper = require('src/mappers/http-error-mapper');
+const GrpcError = require('grpc-error');
+const status = require('src/enums/grpc-status.enum');
```

Comment:
Since we've got `grpc` as a `devDependency` and this is a test file (not available to projects using the package), WDYT about switching this with:

```suggestion
const { status } = require('grpc');
```

so that we could verify that what we're mapping to e.g. `UNKNOWN` is what `grpc` maps to as well?

In other test files as well.

Reply:
Yes, agree with it to make sure the statuses are being correctly mapped.